core/shared/src/main/scala/cats/parse/Parser.scala

```diff
@@ -1072,6 +1072,13 @@ object Parser {
 
   /** Repeat 1 or more times with a separator
     */
+  def repSep[A](p1: Parser[A], sep: Parser0[Any]): Parser[NonEmptyList[A]] =
+    repSep(p1, min = 1, sep)
+
+  /** Repeat `min` or more times with a separator, at least once.
+    *
+    * @throws IllegalArgumentException if `min <= 0`
```

Comment:
🎨 unfortunately scaladoc will not like this. It will need to be `java.lang.IllegalArgumentException`

Reply:
fixed